HtmlBuilderCore/Elements/HtmlHeader.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HtmlBuilderCore.Elements
+{
+    public class HtmlHeader : IHtmlElement
+    {
+
+        public string content_string { get; set; }
```

Comment:
Make anything public cased like this `ContentString` and something not public like this : `contentString`

Even `size_Of_string `=> `SizeOfString`

Not you can right click a variable in visual studio and find Refactor -> Rename and it will rename all usages correctly.